lib/overcommit/hook/pre_commit/es_lint.rb

```diff
@@ -12,19 +12,30 @@ module Overcommit::Hook::PreCommit
   #     enabled: true
   #     command: ['npm', 'run', 'lint', '--', '-f', 'compact']
   #
+  # If ESLint returns some errors, you can bindly pass by
+  # setting `bindly_pass` option to true.
```

Comment:
Can we elaborate on the conditions where this is useful? (concrete examples are very helpful here)